<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gauges Demo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="10,10,1,1" columnstretch="10,1">
   <item row="0" column="0">
    <widget class="QGraphicsViewNoScroll" name="dialView"/>
   </item>
   <item row="0" column="1">
    <widget class="QGraphicsViewNoScroll" name="vertLinearView1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGraphicsViewNoScroll" name="attitudeView"/>
   </item>
   <item row="1" column="1">
    <widget class="QGraphicsViewNoScroll" name="vertLinearView2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGraphicsView" name="horLinearView1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGraphicsView" name="horLinearView2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGraphicsViewNoScroll</class>
   <extends>QGraphicsView</extends>
   <header>qgraphicsviewnoscroll.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
